smartcontract/service/neovm/config.go

```diff
@@ -104,6 +104,7 @@ var (
 	RUNTIME_ADDRESSTOBASE58_NAME     = "Ontology.Runtime.AddressToBase58"
 	RUNTIME_GETCURRENTBLOCKHASH_NAME = "Ontology.Runtime.GetCurrentBlockHash"
 	RUNTIME_VERIFYMUTISIG_NAME       = "Ontology.Runtime.VerifyMutiSig"
+	RUNTIME_ASSERT_NAME              = "Ontology.Runtime.Assert"
```

Comment:
Assert api maybe confused with the python original keyword "assert".  suguest OntAssert

Reply:
ok